pages/services/beta-kafka-zookeeper/2.1.0-3.4.11-beta/api-reference/index.md

```diff
@@ -43,29 +43,29 @@ Use the `/plans` endpoint to list the plans configured for the service.
 ### HTTP example
 
 ```shell
-curl -H "Authorization:token=$auth_token" <dcos_url>/service/kafka-zookeeper/v1/plans/
+curl -H "Authorization:token=$auth_token" <dcos_url>/service/beta-kafka-zookeeper/v1/plans/
 ```
 
 ### CLI example
 
 ```shell
-dcos kafka-zookeeper plan list
+dcos beta-kafka-zookeeper plan list
 ```
 
 ## Plan status
 
-Add the name of the plan to your API request to see the status of a particular plan.
+Add the name of the plan to your API request to see its status.
 
 ### HTTP example
 
 ```shell
-curl -H "Authorization:token=$auth_token" <dcos_url>/service/kafka-zookeeper/v1/plans/deploy
+curl -H "Authorization:token=$auth_token" <dcos_url>/service/beta-kafka-zookeeper/v1/plans/deploy
 ```
 
 ### CLI example
 
 ```shell
-dcos kafka-zookeeper plan status deploy
+dcos beta-kafka-zookeeper plan status deploy
 ```
```

Comment:
this should be 
dcos beta-kafka-zookeeper --name=kafka-zookeeper plan status deploy